examples/cndpfwd/l3-fwd.c

```diff
@@ -0,0 +1,117 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2022 Intel Corporation.
+ */
+
+#include <cne_fib.h>              // For fib_create, fib_add...
+#include <cne_inet4.h>            // for inet_mtoh64 and inet_h64tom
+#include <net/cne_ether.h>        // for ether_addr
+
+#include "main.h"
+
+#define FIB_RULES_IP   0 /**< JSON index for IP address in a single entry */
```

Comment:
Not really a JSON index. Maybe /**< Index to IP address in parsed string array */, ... or something like it.